src/config.ts

```diff
@@ -507,6 +507,9 @@ function assertEnvVarType(name: string, value: any, expectedType: EnvVarType): a
         case EnvVarType.Integer:
             try {
                 returnValue = parseInt(value, 10);
+                if (isNaN(returnValue)) {
+                    throw new Error();
```

Comment:
```suggestion
                    throw new Error(`${name} must be a valid integer, found ${value}.`);
```

Reply:
It's wrapped in a catch below which throws with an appropriate message, I don't think the message here has any impact.